charts/nexus3/values.yaml

```
@@ -356,7 +357,9 @@ config:
   #         expiration: 3
   #       #bucketSecurity:  # If not using Instance Profile
   #       #  accessKeyId: access-key-id
-  #       #  secretAccessKey: secret-access-key
+  #     password:
+  #       secret: test-blob
+  #       key: password
```

```suggestion
  #     secretAccessKey:
  #       secret: test-blob
  #       key: secret-access-key
```

Based on the other secret implementations I think this would be more idiomatic.

---

Even better. The comment should get updated too, I assume.